### DISCUSSION TITLE
Vanillatweaks resourcepacks not being downloaded

### DISCUSSION THREAD

**Discussion by indykoning** [itzg/docker-minecraft-server#2080] [Q&A]
I'm trying to use a resourcepack via the vanillatweaks sharecodes, and the file gets downloaded correctly and as expected.

however when i log into the server i am not prompted about the resource pack, when i check the server.properties i do indeed see that no resourcepack is added.

I see no example in the current readme for extra steps i need to take for resourcepacks to work with Vanillatweaks but i'm guessing i'm missing something.

How can i make sure my Vanillatweaks resourcepack is being used when logging into the server?

**Comment by itzg** [comment]
You will need to further configure that by using

https://github.com/itzg/docker-minecraft-server#custom-server-resource-pack

**Reply by ianmiller347** [reply]
updated link since the redirects from the documentation on github dont work post-migration: https://docker-minecraft-server.readthedocs.io/en/latest/configuration/server-properties/#custom-server-resource-pack

**Comment by dazoe** [comment]
What is the benefit of using the Vanillatweaks resource pack share code? As far as I know the server doesn't do anything with it.
Also It would be nice if the docker image would automatically set RESOURCE_PACK and RESOURCE_PACK_SHA1 when using a share code.

Edit: just found #1740

**Comment by indykoning** [comment]
My personal solution has been to use Polymer
https://polymer.pb4.eu/latest/user/resource-pack-hosting/

While hosting itself is much simpler than this mod, this allows for resource pack merging with other resource packs I want to use
